client/.env

```diff
@@ -0,0 +1 @@
+const BEARER_TOKEN = '387264970d8f2dd42fcf204e92d2e85efa72cf8e84e792c7d642357801cda95d6721af82efdf76b78eda6dd8b1aea5106793727a68fdd8d4df1f238ad39aebb89b9137e4284b86368f4fd49d568a1ecb36ff1702dcf0e2099198fe03ddacbd722516b9eed80b0d7e0586cdeae0061a0d56d06dd61bb469e3c3bbe2dfb4b623e5'
```

Comment:
I don't think you're using this anywhere and you're also not exporting it, so this file is obsolete I guess?